mycroft/tts/tts.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, queue):
         self._terminated = False
         self._processing_queue = False
         self.enclosure = None
+        self.p = None
```

Comment:
Use more descriptive name for this variable.

Reply:
p is something that existed already, it was just not set up so it was guaranteed to exist. It's not in the scope of this PR to change the part of the code